tests/FINDOLOGIC/Export/Tests/DataHelperTest.php

```diff
@@ -13,8 +13,8 @@ class DataHelperTest extends TestCase
     /**
      * @dataProvider emptyValueProvider
      *
-     * @param $value string|int value that should be checked.
-     * @param $shouldCauseException bool should an exception be caused by given parameter.
+     * @param string|int $value Value that should be checked.
+     * @param bool $shouldCauseException Whether the value should cause an exception or not.
      */
     public function testEmptyValueDetectsEmptyStringsOnly($value, $shouldCauseException)
```

Comment:
Type of `$shouldCauseException` is missing.

Reply:
Fixed